src/routes/console/project-[project]/messaging/wizard/emailFormList.svelte

```diff
@@ -9,41 +31,60 @@
         InputText,
         InputTextarea
     } from '$lib/elements/forms';
+
     import { Pill } from '$lib/elements';
     import { CustomId, Modal } from '$lib/components';
     import { user } from '$lib/stores/user';
     import { clickOnEnter } from '$lib/helpers/a11y';
-    import { ID, MessageType, MessagingProviderType } from '@appwrite.io/console';
+    import { ID } from '@appwrite.io/console';
     import { sdk } from '$lib/stores/sdk';
+    
+    import { MessageType, MessagingProviderType } from '@appwrite.io/console';
+    import {page} from '$app/stores'
+    import Alert from '$lib/components/alert.svelte';
 
     let showCustomId = false;
     let showTest = false;
     let selected = 'self';
+
     let otherEmail = '';
 
     async function sendTestEmail() {
         const email = selected === 'self' ? $user.email : otherEmail;
         console.log(email);
 
-        // TODO: replace with test method
-        sdk.forProject.messaging.createEmail(
-            ID.unique(),
-            $messageParams[MessagingProviderType.Email]?.subject || undefined,
-            $messageParams[MessagingProviderType.Email]?.content || undefined,
-            $messageParams[MessagingProviderType.Email]?.topics || [],
-            $messageParams[MessagingProviderType.Email]?.users || [],
-            $messageParams[MessagingProviderType.Email]?.targets || [],
-            undefined,
-            undefined,
-            MessageType.Processing,
-            $messageParams[MessagingProviderType.Email]?.html || false,
-            undefined
-        );
+        createEmailMessage({
+            topics: $messageParams[MessagingProviderType.Email]?.topics || [],
+            targets: $messageParams[MessagingProviderType.Email]?.targets || [],
+            status: MessageType.Processing,
+            messageId: ID.unique(),
+            // TODO: properly handle the test email address
+            users: ['steven'],
+            subject: $messageParams[MessagingProviderType.Email]?.subject || '',
+            content: $messageParams[MessagingProviderType.Email]?.content || '',
+            html: $messageParams[MessagingProviderType.Email]?.html || false
+        });
     }
 
     $: otherEmail = selected === 'self' ? '' : otherEmail;
+
+    const providers = () => {
+      return $page.data.providers.providers.filter((provider) => provider.type === 'email');
```

Comment:
Can we use the enum instead of an `'email'` string?

Also, I'm slightly concerned about this approach on the small chance someone has a full page of providers that doesn't match for their first page of providers.